great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -88,6 +86,10 @@ def metric_domain_kwargs(self) -> Optional[Union[str, dict]]:
     def metric_value_kwargs(self) -> Optional[Union[str, dict]]:
         return self._metric_value_kwargs
 
+    @metric_value_kwargs.setter
+    def metric_value_kwargs(self, value: Optional[Union[str, dict]]):
+        self._metric_value_kwargs = value
```

Comment:
Please annotate the return.

Reply:
@cdkini -- nice catch!  DONE (in next commit).  Thanks!